### PR TITLE
CSAF

### PR DESCRIPTION
**Pull request by tschmidtb51**
- addresses part of oasis-tcs/csaf#38
- correct mistake in schema found with ajv

### CODE REVIEW

File: csaf_2.0/json_schema/csaf_json_schema.json [hunk 1]
```diff
@@ -1250,7 +1250,7 @@
               "title": "Score",
               "description": "specifies information about (at least one) score of the vulnerability and for which products the given value applies.",
               "type": "object",
-              "minItems": 2,
+              "minProperties": 2,
```
Comment (sthagen) [inline csaf_2.0/json_schema/csaf_json_schema.json:1253]: 👀

**Review by sthagen — APPROVED** [review]
Good catch.

File: csaf_2.0/json_schema/csaf_json_schema.json (hunk 1 above)
Comment (santosomar) [reply csaf_2.0/json_schema/csaf_json_schema.json:1253]: Oh wow! Great catch indeed! Thank you so much!

**Review by santosomar — APPROVED** [review]

File: csaf_2.0/json_schema/csaf_json_schema.json (hunk 1 above)
Comment (tschmidtb51) [reply csaf_2.0/json_schema/csaf_json_schema.json:1253]: It was actually searching for the mistake I (then) corrected in #310... This was more accidentally.

**Pull request merged**